back/app/services/survey_results_generator_service.rb

```diff
@@ -39,29 +39,35 @@ def visit_multiline_text(field)
     answers = inputs
       .select("custom_field_values->'#{field.key}' as value")
       .where("custom_field_values->'#{field.key}' IS NOT NULL")
-      .map(&:value)
+      .map do |answer|
+        { answer: answer.value }
+      end
     answer_count = answers.size
     {
       inputType: field.input_type,
       question: field.title_multiloc,
       required: field.required,
       totalResponses: answer_count,
-      customFieldId: field.id
+      customFieldId: field.id,
+      textResponses: answers
```

Comment:
I've added this `textResponses` property to all question types that can have text answers. This approach should work well for the "Other" option, as well. I considered having a separate endpoint for this but decided against it as I am using a virtualized list to display the answers on the front-end, meaning even if they are a really large number, we should not see a performance issue.